.github/workflows/knative-style.yaml

```diff
@@ -188,11 +199,17 @@ jobs:
           # Don't fail because of misspell
           set +o pipefail
           # Lint exclude rule:
-          #  - nothing in vendor/
+                    #  - nothing in vendor/
```

Comment:
This indent seems off but not sure why since:
https://github.com/knative-sandbox/.github/blob/master/workflow-templates/knative-style.yaml#L202

Seems to be just fine.